src/scenes/Oracle/components/PredictionAlert/index.js

```diff
@@ -0,0 +1,48 @@
+import React, { PropTypes } from 'react';
+import Button from 'material-ui/Button';
+import Dialog, {
+  DialogActions,
+  DialogContent,
+  DialogContentText,
+  DialogTitle,
+} from 'material-ui/Dialog';
+
+class PredictionAlert extends React.PureComponent {
+  render() {
+    const { requestReturn } = this.props;
+
+    return (
+      <Dialog
+        open={requestReturn}
+        onClose={this.handleAlertClose}
+        aria-labelledby="alert-dialog-title"
+        aria-describedby="alert-dialog-description"
+      >
+        <DialogTitle id="alert-dialog-title">{requestReturn && requestReturn.result ? 'Success!' : 'Oops, something went wrong'}</DialogTitle>
+        <DialogContent>
+          <DialogContentText id="alert-dialog-description">
+            {requestReturn && requestReturn.result ?
+              'The transaction is broadcasted to blockchain. You can view details at https://testnet.qtum.org/tx/'.concat(requestReturn && requestReturn.result.txid).concat('!') :
+              requestReturn && requestReturn.error
+            }
+          </DialogContentText>
+        </DialogContent>
+        <DialogActions>
+          <Button onClick={this.handleAlertClose} color="primary">
+            OK
+          </Button>
+        </DialogActions>
+      </Dialog>
+    );
+  }
+
+  handleAlertClose() {
+    window.location.reload();
```

Comment:
Don't think we need to force a reload on the page. This kinda makes it hard to debug too since the console gets cleared.

The transactions seem like there is something wrong with them.

Reply:
Can we just clear out certain values in the state to reset everything?